content/source/docs/cloud/users-teams-organizations/organizations.html.md

```diff
@@ -43,61 +44,95 @@ You can view and manage an organization's settings by clicking the "Settings" li
+
+The list of users is split into two tabs: one for currently active users, and one for invited users who haven't accepted their invitations yet. If the lists are large, you can filter them by username or email using the search field at the top. For active users, the list includes usernames, email addresses, avatar icons, two-factor authentication status, and current team memberships.
+
+![Screenshot: the users list, showing information for the currently active users.](./images/org-users.png)
+
+To invite a user to an organization, click the "Invite a user" button on the "Users" page. You will be asked to provide an email address and an optional list of teams. If the user accepts the emailed invitation, they will be automatically added to the specified teams.
+
+![Screenshot: the "Invite a user" dialog, which includes fields for email address and teams.](./images/org-users-invite.png)
+
+User invitations are always sent by email; you cannot invite someone using their Terraform Cloud username.
+
+-> **Note:** All permissions in Terraform Cloud are managed via teams. Although users can join an organization without belonging to any teams, they won't be able to do anything until they are also added to a team.
 
 ### Teams
 
-The teams page allows organization owners to manage the organization's teams, including creating and deleting teams, managing membership, and generating team API tokens.
+-> **Note:** Team management is a paid feature, available as part of the **Team** upgrade package. [Learn more about Terraform Cloud pricing here](https://www.hashicorp.com/products/terraform/pricing/).
+
+The teams page is shown to all users in an organization.
 
-Users who aren't organization owners can view the list of teams they belong to and the membership of those teams. They can't edit teams or view any teams they don't belong to.
+Organization owners can use this page to create and delete teams, manage team membership, and manage team API tokens. Note that users can only be added to teams after they have received and accepted an invitation to the organization.
+
+Non-owners can view the list of teams they belong to, view the membership of those teams, and manage team API tokens for those teams. They can't edit team memberships or view any teams they don't belong to.
 
 See [Teams][] for more information.
 
-### VCS Provider
+### VCS Providers
 
-The VCS provider page is used for setting up VCS access for Terraform Cloud. See [Connecting VCS Providers](../vcs/index.html) for more information.
+The VCS providers page is used for setting up VCS access for Terraform Cloud. See [Connecting VCS Providers](../vcs/index.html) for more information.
 
 ### API Tokens
 
 Organizations can have a special API token that is not associated with a specific user or team. See [API Tokens](./api-tokens.html) for more information.
 
 ### Authentication
 
+-> **Note:** This page is not available in free organizations.
```

Comment:
Yeah... it's because it's missing due to a bug, not due to a conscious product decision. 😅 Maybe I should just remove this and assume we'll fix it soonish, but at the time I wrote this that was still in flux.